packages/events/src/EventSystem.ts

```diff
@@ -45,6 +77,38 @@ export class EventSystem implements ISystem<EventSystemOptions>
         ],
     };
 
+    private static _defaultEvents: Record<'move' | 'globalMove' | 'click' | 'wheel', boolean> = {
+        move: true,
+        globalMove: true,
+        click: true,
+        wheel: true,
+    };
+
+    /**
+     * The event types that are enabled by the EventSystem
+     * @type {object}
+     * @readonly
+     * @since 7.2.0
+     * @property {boolean} move - Enables pointer events associated with pointer movement:
+     * - `pointermove` / `mousemove` / `touchmove`
+     * - `pointerout` / `mouseout`
+     * - `pointerover` / `mouseover`
+     * @property {boolean} globalMove - Enables global pointer move events:
+     * - `globalpointermove`
+     * - `globalmousemove`
+     * - `globaltouchemove`
+     * @property {boolean} click - Enables pointer events associated with clicking:
+     * - `pointerup` / `mouseup` / `touchend` / 'rightup'
+     * - `pointerupoutside` / `mouseupoutside` / `touchendoutside` / 'rightupoutside'
+     * - `pointerdown` / 'mousedown' / `touchstart` / 'rightdown'
+     * - `click` / `tap`
+     * @property {boolean} wheel - Enables wheel events.
+     */
+    public static get defaultEvents()
```

Comment:
Can we re-think static usage here? You're only referencing this internally.

For instance, if you wanted to change something _after_ construction, you'd do this:
```ts
import { Renderer, EventSystem } from 'pixi.js';

const renderer = new Renderer();
EventSystem.defaultEvents.wheel = false;
```

I think this is better:

```ts
import { Renderer } from 'pixi.js';

const renderer = new Renderer();
renderer.events.features.wheel = false; // or another name for "features"
```

The benefit is that it allow us to isolate Renderer instance of those people creating multiple renderers at once.

Reply:
That said, I think `_defaultEvents` (object above) should be public, but internally we should create an object on the instance.

Reply:
I've removed the static member in favour for `private _features`

I don't think we should be exposing these options outside of the renderer initialisation otherwise we have to handle adding / removing the DOM events which currently only happens on `init`

Reply:
That's fine with me. We can easily add that later, better to keep the public API small as possible